HtmlWork/ex3.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html >
+<table>
+    <caption>My favorite movies</caption>
+    <tr>
+      <th>The name of the film</th>
+      <th>Description</th>
+      <th>Poster</th>
+    </tr>
+    <tr>
+      <td>The Shawshank Redemption (1994)</td>
+      <td>In 1947, Andy Dufresne is sentenced to life in Shawshank State Penitentiary for the murders of his wife and her lover, despite his claims of innocence. Over the following two decades, he befriends a fellow prisoner, contraband smuggler Ellis "Red" Redding, and becomes instrumental in a money-laundering operation led by the prison warden Samuel Norton.</td>
+<td>
+    <img src="file:///C:/Code/IntrotoGit/HtmlWork/src/poster_Shawshank_Redemption.webp" alt="Poster Shawshank">
```

Comment:
Make this relative, otherwise will not work for other people.

```suggestion
    <img src="poster_Shawshank_Redemption.webp" alt="Poster Shawshank">
```